examples/tensorflow-profiling-examples/trainer-deep-profiling/task.py

```diff
@@ -0,0 +1,84 @@
+import argparse
+import json
+import os
+
+import model
+
+
+def parse_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--bucket',
+                        help='GCS path to data',
+                        required=True)
+    parser.add_argument('--output_dir',
+                        help='GCS loc to write checkpoints and export models',
+                        required=True)
+    parser.add_argument('--train_steps',
+                        help='Steps to run the training job for.',
+                        type=int,
+                        default=10000)
+    parser.add_argument('--eval_int',
+                        help='Evaluation interval in seconds',
+                        type=int,
+                        default=200)
+    parser.add_argument('--pattern',
+                        help='Specify a pattern that has to be in input files. For example 00001-of will process only one shard',  # pylint: disable=C0301
+                        default='of')
+    parser.add_argument('--job-dir',
+                        help='we ignore it, but it is required',
+                        default='junk')
+    parser.add_argument('--eval_steps',
+                        type=int,
+                        default=None)
+    parser.add_argument('--batch_size',
+                        type=int,
+                        default=512)
+    parser.add_argument('--n_embeds',
+                        type=int,
+                        default=3)
+    parser.add_argument('--dropout_rate',
+                        type=float,
+                        default=0.1)
+    parser.add_argument('--learning_rate',
+                        type=float,
+                        default=0.001)
+    parser.add_argument('--first_layer_size',
+                        type=int,
+                        default=64)
+    parser.add_argument('--num_layers',
+                        type=int,
+                        default=2)
+    args = parser.parse_args()
+    return args.__dict__
```

Comment:
I think you should use vars(args) instead of calling the internal __dict__ method.

Reply:
ack